ods_ci/tests/Tests/400__ods_dashboard/413__ods_dashboard_user_mgmt.robot

```diff
@@ -49,28 +49,29 @@ Verify If Unauthorized User Can Not Change The Permission
 Verify Unauthorized User Is Not Able To Spawn Jupyter Notebook
     [Documentation]    Verify unauthorized User Is Not Able To Spawn Jupyter
     ...     Notebook , user should not see a spawner if the user is not in allowed Groups
+    ...     Note: this test creates users/groups via ODH Dashboard UI in
+    ...     `Settings -> User management` section. There is an another test that changes
+    ...     users/groups via `oc adm groups` command,see: `Verify User Can Set Custom RHODS Groups`
+    ...     in ods_ci/tests/Tests/500__jupyterhub/jupyterhub-user-access.robot
     [Tags]  ODS-1680
     ...     Tier1
     ...     Sanity
     Launch Dashboard And Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
     Clear User Management Settings
-    Add OpenShift Groups To Data Science Administrators    rhods-users
-    Add OpenShift Groups To Data Science User Groups       rhods-users
+    Add OpenShift Groups To Data Science Administrators    rhods-admins
+    Add OpenShift Groups To Data Science User Groups       rhods-admins
     Save Changes In User Management Setting
-    AdminGroups In OdhDashboardConfig CRD Should Be        rhods-users
-    AllowedGroups In OdhDashboardConfig CRD Should Be      rhods-users
-    Reload Page
-    ${version_check}=    Is RHODS Version Greater Or Equal Than    1.20.0
-    IF    ${version_check} == True
-        ${status}=    Run Keyword And Return Status     Launch Jupyter From RHODS Dashboard Link
-        Run Keyword And Continue On Failure    Should Be Equal    ${status}    ${FALSE}
-        Run Keyword And Continue On Failure    Page Should Contain    Access permissions needed
-        Run Keyword And Continue On Failure    Page Should Contain    ask your administrator to adjust your permissions.
-    ELSE
-        Menu.Navigate To Page    Applications    Enabled
-        Launch Jupyter From RHODS Dashboard Link
-        Run Keyword And Continue On Failure   Verify Jupyter Access Level   expected_result=none
-    END
+    AdminGroups In OdhDashboardConfig CRD Should Be        rhods-admins
+    AllowedGroups In OdhDashboardConfig CRD Should Be      rhods-admins
+    Logout From RHODS Dashboard
+    Launch Dashboard    ocp_user_name=${TEST_USER_4.USERNAME}    ocp_user_pw=${TEST_USER_4.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER_4.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}
+    ...    browser=${BROWSER.NAME}    browser_options=${BROWSER.OPTIONS}
+    ...    expected_page=${NONE}      wait_for_cards=${FALSE}
+    # ${status}=    Run Keyword And Return Status     Launch Jupyter From RHODS Dashboard Link
+    # Run Keyword And Continue On Failure    Should Be Equal    ${status}    ${FALSE}
+    Run Keyword And Continue On Failure    Page Should Contain    Access permissions needed
+    Run Keyword And Continue On Failure    Page Should Contain    ask your administrator to adjust your permissions.
```

Comment:
You open a new browser but never close it, can you add a final step to close the browser and switch the context to the previous one?

Reply:
Uff, very well spotted, Luca! Thanks for this. I'll update.